release-scripts/prepare-release-formula.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [ -z "$RELEASE_TAG" ]
+then
+  echo "\$RELEASE_TAG is empty"
+  exit 1
+fi
+
+version=${RELEASE_TAG:1}
+
+function install_mvn_dependency {
+	rm -rf $1
+	git clone git@github.com:confluentinc/$1.git
+	cd $1
+	git checkout $RELEASE_TAG
+	mvn clean install -DskipTests
+	cd -
+}
+
+cd /tmp
+
+rm -rf kafka
+git clone git@github.com:confluentinc/kafka.git
+cd kafka
+git checkout $RELEASE_TAG
+gradle
+./gradlew clients:install connect:api:install connect:runtime:install install_2_11
```

Comment:
Why do we need the Connect runtime?

Reply:
It's a dependency of `license-file-generator`.